File: compiled_languages/swift/classes.swift
import Cocoa

//--------------- class
class Person {
    var age: Int

    init(age: Int) {
       self.age = age
    }

    func addFive(c: Int) -> Int 
    {
        return c + 5
    }
}

//--------------- instantiation
let brian = Person(age: 27)
print("brian is \(brian.age)\n")
print("brian: \(brian.addFive(10))\n")

File: _scripting_languages/Python/classes.py
class Person:
    "this is a person"
    personCount = 0 # class attribute
    
    def __init__(self, name, age):
        self.name = name 
        self.age = age 
        Person.personCount += 1 
    
    def sayHello(self):
        print("hello there! my name is: " + self.name)
        
    def sayAge(self):
        print("I am " + str(self.age) + " years old.")


brian = Person("Brian Spinos", 28)
brian.sayHello()
brian.sayAge()

erich = Person("erich Spinos", 25)
erich.sayHello()
erich.sayAge()

print(Person.personCount) # 2
